tool/tsh/tsh.go

```diff
@@ -656,6 +657,7 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 	login.Arg("cluster", clusterHelp).StringVar(&cf.SiteName)
 	login.Flag("browser", browserHelp).StringVar(&cf.Browser)
 	login.Flag("kube-cluster", "Name of the Kubernetes cluster to login to").StringVar(&cf.KubernetesCluster)
+	login.Flag("verbose", "Show extra status information").Short('v').BoolVar(&cf.Verbose)
```

Comment:
I'm not 100% sure if `tsh login` needs to have the verbose flag. But I guess maybe it's fine in terms of consistency, since it shows the same info as `tsh status` and `tsh status` does support this flag?